msvc_build.bat

```diff
@@ -34,8 +34,8 @@ REM BUILD_64 and BUILD_FOR_WINSTORE are used in CMakeLists.txt
 SET BUILD_FOR_WINSTORE=OFF
 SET BUILD_64=ON
 
-SET BUILD_FOLDER=msvc.build
-SET INSTALL_FOLDER=msvc.install
+SET BUILD_FOLDER=msvc.build2
+SET INSTALL_FOLDER=msvc.install2
```

Comment:
Don't do this. You may locally, bit should not commit such a change

Reply:
Ah, a mistake...

Reply:
also the commit title might better be "fix format issues + collect_artifacts"

Reply:
Oh, I thought it had to be in the beginning... Thanks and I'll push again.